### ISSUE TITLE
Translatable not join to Translation

### ISSUE THREAD

**Issue by uginroot** [KnpLabs/DoctrineBehaviors#532]
# What I've done

## Create new symfony project `knp_test`
```bash
X:\>symfony new knp_test --full
*   (running Y:\php\composer\composer.phar create-project symfony/website-skeleton X:\knp_test)
 |
* Setting up the project under Git version control
  (running git init X:\knp_test)

 [OK] Your project is now ready in X:\knp_test
```
## Install knplabs/doctrine-behaviors
```bash
X:\knp_test>composer require knplabs/doctrine-behaviors
Using version ^2.0 for knplabs/doctrine-behaviors
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Restricting packages listed in "symfony/symfony" to "5.0.*"
Package operations: 5 installs, 0 updates, 0 removals
  - Installing ramsey/uuid (3.9.3): Loading from cache
  - Installing nette/utils (v3.1.1): Loading from cache
  - Installing nette/finder (v2.5.2): Loading from cache
  - Installing symplify/package-builder (v7.2.4): Loading from cache
  - Installing knplabs/doctrine-behaviors (v2.0.1): Loading from cache
Writing lock file
Generating autoload files
ocramius/package-versions: Generating version class...
ocramius/package-versions: ...done generating version class
Executing script cache:clear [OK]
Executing script assets:install public [OK]
```
## Create entity `Category`
```php
namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;
use Knp\DoctrineBehaviors\Contract\Entity\TranslatableInterface;
use Knp\DoctrineBehaviors\Model\Translatable\TranslatableTrait;

/**
 * @ORM\Entity(repositoryClass="App\Repository\CategoryRepository")
 */
class Category implements TranslatableInterface
{
    use TranslatableTrait;

    /**
     * @ORM\Id()
     * @ORM\GeneratedValue()
     * @ORM\Column(type="integer")
     */
    private $id;

    public function getId(): ?int
    {
        return $this->id;
    }
}
```

## Create entity `CategoryTranslation`
```php
namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;
use Knp\DoctrineBehaviors\Contract\Entity\TranslationInterface;
use Knp\DoctrineBehaviors\Model\Translatable\TranslationTrait;

/**
 * @ORM\Entity
 */
class CategoryTranslation implements TranslationInterface
{
    use TranslationTrait;

    /**
     * @ORM\Id()
     * @ORM\GeneratedValue()
     * @ORM\Column(type="integer")
     */
    private $id;

    /**
     * @ORM\Column(type="string", length=255)
     */
    private $name;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getName(): ?string
    {
        return $this->name;
    }

    public function setName(string $name): self
    {
        $this->name = $name;

        return $this;
    }
}
```
## Make migration
```bash
X:\knp_test>php bin/console make:migration

  Success!

 Next: Review the new migration "src/Migrations/Version20200317093320.php"
 Then: Run the migration with php bin/console doctrine:migrations:migrate
 See https://symfony.com/doc/current/bundles/DoctrineMigrationsBundle/index.html
```
## Call doctrine:migration:migrate command
```bash
X:\knp_test>php bin/console doctrine:migration:migrate

                    Application Migrations

WARNING! You are about to execute a database migration that could result in schema changes and data loss. Are you sure you wish to continue? (y/n)y
Migrating up to 20200317093320 from 0

  ++ migrating 20200317093320

     -> CREATE TABLE category (id INT AUTO_INCREMENT NOT NULL, PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB
     -> CREATE TABLE category_translation (id INT AUTO_INCREMENT NOT NULL, name VARCHAR(255) NOT NULL, PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB

  ++ migrated (took 460.5ms, used 12M memory)

  ------------------------

  ++ finished in 463.6ms
  ++ used 12M memory
  ++ 1 migrations executed
  ++ 2 sql queries
```
## Table `category_translation`
Columns available: `id`, `name`
Expected columns: `id`, `name`, `locale`, `translatable_id`

## Tests knplabs/doctrine-behaviors on my system
```bash
X:\DoctrineBehaviors>"vendor/bin/phpunit"
PHPUnit 8.5.2 by Sebastian Bergmann and contributors.

................................................................. 65 / 69 ( 94%)
....                                                              69 / 69 (100%)

Time: 10.55 seconds, Memory: 62.00 MB

OK (69 tests, 192 assertions)
```
## Enveroment
PHP 7.4.1 (cli) (built: Dec 17 2019 19:24:02) ( ZTS Visual C++ 2017 x64 )
Ver 15.1 Distrib 10.4.11-MariaDB, for Win64 (AMD64)

**Comment by uginroot** [comment]
I think this #525 is related to this. Moreover, all the data is written to the tables, but there is no reference to the parent entity in the translations.

**Comment by uginroot** [comment]
I'm an idiot...
I'm not register bandle in `config/bundles.php`.
Why you can’t do this automatically as in other extensions?

**Comment by TomasVotruba** [comment]
I think it needs some Flex recipe. I'm not sure how to work with them.
Could you create one for this bundle please?

**Comment by uginroot** [comment]
I will try to implement this

**Comment by TomasVotruba** [comment]
Thank you

**Comment by uginroot** [comment]
#534 Changes for correct bundle work

**Comment by TomasVotruba** [comment]
GitHub tip, if you tipe "closes #532" in the pull-request description, it links this PR to this issues and also closes the issue on merge :+1:
